examples/substrate-asset-transfer/src/transfer.ts

```diff
@@ -0,0 +1,86 @@
+import { Keyring } from "@polkadot/keyring";
+import { ApiPromise, WsProvider } from '@polkadot/api';
+import {
+  Domain,
+  Environment,
+  Fungible,
+  Resource,
+  SubstrateAssetTransfer,
+  SubstrateParachain,
+  Transfer,
+} from "@buildwithsygma/sygma-sdk-core";
+
+export const ROCOCO_CHAIN_ID = 5231
+export const SEPOLIA_CHAIN_ID = 11155111;
+
+
+export async function substrateTransfer(): Promise<void> {
+
+
+  const keyring = new Keyring({ type: "sr25519" });
+  // Make sure to fund this accoubnt with both native and asset tokens
+  // Account address: 5FNHV5TZAQ1AofSPbP7agn5UesXSYDX9JycUSCJpNuwgoYTS
+  const mnemonic =
+    "zoo slim stable violin scorpion enrich cancel bar shrug warm proof chimney";
+  const account = keyring.addFromUri(mnemonic);
+
+  const wsProvider = new WsProvider('ws://127.0.0.1:9944');
+  const api = await ApiPromise.create({ provider: wsProvider });
+
+  const assetTransfer = new SubstrateAssetTransfer();
+
+  await assetTransfer.init(
+    api,
+    Environment.TESTNET,
+    SubstrateParachain.ROCOCO_PHALA
```

Comment:
Can we make this a required param please and remove the default `LOCAL` so nobody makes that mistake please.